internals/secrethub/service_init.go

```diff
@@ -117,7 +117,9 @@ func (cmd *ServiceInitCommand) Run() error {
 func (cmd *ServiceInitCommand) Register(r Registerer) {
 	clause := r.Command("init", "Create a new service account attached to a repository.")
 	clause.Arg("path", "The service account is attached to the repository in this path and when used together with --permission, an access rule is created on the directory in this path.").Required().SetValue(&cmd.path)
-	clause.Flag("desc", "A description for the service").StringVar(&cmd.description)
+	clause.Flag("description", "A description for the service").StringVar(&cmd.description)
```

Comment:
`A description (max 144 chars) for the service so others will recognize it.`

Reply:
The description of a service is not limited to 144 chars.

Reply:
It's limit is 2048 chars

Reply:
How about `A description for the service to recognize it.`? It's not only for others, as it can be useful for yourself as well.

Reply:
Ah ok. Euhmm, it's good to mention `others` here because it frames it as a collaborative effort. I think once about stuff for myself (or not at all), but for others I think twice about naming things so they make sense.

So I'd say: `A description for the service so others will recognize it.`